.github/workflows/ci.yml

```diff
@@ -40,8 +40,6 @@ jobs:
         run: cargo build
       - name: Ensure C headers are up to date
         run: |
-          cargo install cbindgen
-          script/cbindgen
```

Comment:
_TLDR_ Yes, but it's not as good in practice as I hoped, so I'm taking this out to rethink it.

That is what is happening here. But I'm happy you call this out, as I forgot that change was in this PR. In practice I haven't been very happy with how that works out. In particular, the `build.rs` is run _before_ the regular build, so if there are issues, you sometimes get errors from cbindgen instead of rustc. And if cbindgen fails, the compiler isn't even run, so you're stuck with the subpar error messages generated by cbindgen. Therefore, I'm going to take this change out and see if I can redo it in another way, but I'll make a separate PR for that.